scripts/tests/chiptest/test_definition.py

```diff
@@ -208,15 +217,37 @@ def Run(self, runner, apps_register, paths: ApplicationPaths, pics_file: str):
 
         try:
             if self.target == TestTarget.ALL_CLUSTERS:
-                app_cmd = paths.all_clusters_app
+                target_app = paths.all_clusters_app
             elif self.target == TestTarget.TV:
-                app_cmd = paths.tv_app
+                target_app = paths.tv_app
             elif self.target == TestTarget.LOCK:
-                app_cmd = paths.lock_app
+                target_app = paths.lock_app
             else:
                 raise Exception("Unknown test target - "
                                 "don't know which application to run")
 
+            # The paths argument is a dataclass where each field contains a list of one application path
+            for field in paths.__dataclass_fields__:
```

Comment:
__dataclass_fields__ seems to be private based on naming. I do not believe it is ok to access them like this.

Reply:
Added a getter in `ApplicationPaths` to make expose this same info